contracts/LaunchpegLens.sol

```diff
@@ -367,6 +399,25 @@ contract LaunchpegLens {
         }
     }
 
+    function _getERC1155SingleBundleData(
+        address launchpeg
+    ) private view returns (ERC1155SingleBundleData memory data) {
+        ERC1155SingleBundle lp = ERC1155SingleBundle(launchpeg);
+        data.tokenSet = lp.tokenSet();
+        data.currentPhase = IBaseLaunchpeg.Phase(uint8(lp.currentPhase()));
+        data.amountForPreMint = lp.amountForPreMint();
```

Comment:
we use amountForAllowlist both for premint and allowlist for Launchpegs. not sure if we want to move to naming convention of amountForPreMint or leave it as amountForAllowlist for frontend convenience, this will need to be custom anyway right?

Reply:
Yeah idk, the whole codebase would need a cleanup tbh, maybe we can keep it the way it is in the meantime